AzureMonitorAgent/agent.py

```diff
@@ -722,10 +778,22 @@ def metrics_watcher(hutil_error, hutil_log):
     # Sleep before starting the monitoring
     time.sleep(sleepTime)
     last_crc = None
+    last_crc_fluent = None
     me_msi_token_expiry_epoch = None
 
     while True:
         try:
+            if os.path.isfile(FluentCfgPath):
```

Comment:
Take a note that it is the agentlauncher's responsibility to detect config file changes. Kennedy is going to add that in next iteration.

Reply:
yeah, we will remove this then